packages/react-html/src/server/components/Html.tsx

```diff
@@ -84,36 +79,28 @@ export default function HTML({
     );
   });
 
-  /* eslint-disable no-undefined */
-  const bodyStyles = hideForInitialLoad ? {display: 'none'} : undefined;
-  /* eslint-enable no-undefined */
-
-  const headDataMarkup = Object.keys(headData).map(id => {
-    return <Serializer key={id} id={id} data={headData[id]} />;
-  });
-
-  const dataMarkup = Object.keys(data).map(id => {
-    return <Serializer key={id} id={id} data={data[id]} />;
-  });
+  const bodyStyles =
+    // eslint-disable-next-line no-process-env
+    process.env.NODE_ENV === 'development' ? {display: 'none'} : undefined;
 
   return (
-    <html lang="en" {...htmlAttributes}>
+    <html lang={locale} {...htmlAttributes}>
       <head>
         {helmet.title.toComponent()}
         {helmet.meta.toComponent()}
         {helmet.script.toComponent()}
         {helmet.link.toComponent()}
 
         {stylesMarkup}
```

Comment:
There are times where we'd want markup before styles. Should we simply rely on Helmet for that use-case?

Reply:
I think we'd rely on helmet, I am not sure what markup would go above styles other than links/ scripts/ meta tags?

Reply:
You're right, thanks!